DZ4_MATH_OPERATIONS/index.js

```diff
@@ -0,0 +1,60 @@
+function sumReducer(state, action) {
```

Comment:
You have done a lot to solve the task, this is good

But, the output is not correct

Such solution will be ok

```
const number1 = Number(prompt('Plz, enter first number'));
const number2 = Number(prompt('Plz, enter second number'));

alert (`
    User entered ${number1} and ${number2}
    - ${number1} + ${number2} = ${number1 + number2}
    - ${number1} - ${number2} = ${number1 - number2}
    - ${number1} * ${number2} = ${number1 * number2}
    - ${number1} / ${number2} = ${number1 / number2}
`);
```